src/Inciter/FluxCorrector.C

```diff
@@ -396,8 +396,8 @@ FluxCorrector::lim( const std::vector< std::size_t >& inpoel,
 //! \param[in] Ul Low order solution
 //! \param[inout] Q The maximum and mimimum unknowns of elements surrounding
 //!   each node
-//! \param[in,out] Limited antidiffusive element contributions scatter-added to
-//!   nodes
+//! \param[in,out] A Limited antidiffusive element contributions scatter-added
```

Comment:
Not a heavy fortran programmer, but what is the distinction between the inout and in,out intent?